public/index.html

```diff
@@ -9,12 +9,12 @@
       name="description"
       content="Prepare your Resume on the Go, even from your Mobile!"
     />
-    <link rel="apple-touch-icon" href="logo192.png" />
+    <!-- <link rel="apple-touch-icon" href="logo192.png" /> -->
     <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!-- <link rel="manifest" href="%PUBLIC_URL%/manifest.json" /> -->
```

Comment:
Why were these lines commented out?

Reply:
these lines was linked with the logo which was of react so I just commented so that when you decide the actual logo then we can remove this comment-out line and add it.

Reply:
Let's leave this the same way. It's better to have this instead of nothing. So closing this as invalid.